packages/embed/src/apple-pay/apple-pay.ts

```diff
@@ -1,11 +1,66 @@
 import { SubjectManager } from 'subjects'
+import { error, log, warn } from 'utils'
+
+export const APPLE_PAY_TIMEOUT = 1000
 
 declare global {
   interface Window {
     ApplePaySession?: ApplePaySession
   }
 }
 
+export const loadApplePaySdk = () => {
```

Comment:
Race condition, so if the first Promise takes too long to resolve (more than 1s), we log a warning and consider the Apple Pay JS SDK as not loaded.

The first Promise:
- Checks that either an Apple Pay session already exists or we have already loaded the script
- If not, it creates the script, attempts to load it and does some checks to see if it was loaded correctly